site/src/Sidecar.jsx

```diff
@@ -0,0 +1,171 @@
+import "./Sidecar.css";
+import CloseIcon from "@mui/icons-material/Close";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import FlightTakeoffIcon from "@mui/icons-material/FlightTakeoff";
+
+function Sidecar({ open, setOpen, map, setVisibleTypes, setActiveThemes }) {
+  const handleParis = () => {
+    setVisibleTypes([
+      "division_area",
+      "boundary",
+      "land",
+      "land_cover",
+      "land_use",
+      "water",
+      "infrastructure",
+      "segment",
+      "connector",
+      "building",
+      "building_part",
+    ]);
+    setActiveThemes(["transportation"]);
+    map.ref.current.jumpTo({
+      center: [2.3417, 48.8552],
+      zoom: 11.73,
+      pitch: 12,
+      bearing: 15.3,
+    });
+  };
+
+  const handleNYC = () => {
+    setVisibleTypes([
+      "boundary",
+      "land",
+      "land_cover",
+      "land_use",
+      "water",
+      "infrastructure",
+      "building",
+      "building_part",
+    ]);
+    setActiveThemes(["buildings"]);
+    map.ref.current.jumpTo({
+      center: [-73.99768, 40.75332],
+      zoom: 14.22,
+      pitch: 60,
+      bearing: 60.6,
+    });
+  };
+  const handleWorld = () => {
+    setVisibleTypes([
+      "land",
+      "land_cover",
+      "land_use",
+      "water",
+      "infrastructure",
+    ]);
+    setActiveThemes(["base"]);
+    map.ref.current.jumpTo({
+      center: [10.3, 24.5],
+      zoom: 1.07,
+      bearing: 0,
+      pitch: 0,
+    });
+  };
+  const handleLondon = () => {
+    setVisibleTypes([
+      "division_area",
+      "boundary",
+      "land",
+      "land_cover",
+      "land_use",
+      "water",
+      "infrastructure",
+      "segment",
+      "building",
+      "building_part",
+      "place",
+    ]);
+    setActiveThemes(["places"]);
+    map.ref.current.jumpTo({
+      center: [-0.091217, 51.514511],
+      zoom: 16.02,
+      bearing: -24,
+      pitch: 50,
+    });
+  };
+  const handleBoston = () => {
```

Comment:
Love the nice declarative style here that you have for each POI - looks really easy to extend and add new ones!

Reply:
On that topic, I loved some of the place suggestions that the team indicated in the slack thread you created- could we add a couple of those here too?

Reply:
I've added athens and hong kong ports. I'm not suite sure how zoomed in we hope to get - stuff like the magic roundabout and rittenhouse square is _super_ zoomed in, so Im not quite sure how valuable that is